app/controllers/gobierto_admin/gobierto_participation/processes/process_contribution_containers_controller.rb

```diff
@@ -42,11 +45,10 @@ def create
             track_create_activity
 
             redirect_to(
-              edit_admin_participation_process_contribution_container_path(process_id: @process.id,
-                                                                           id: @contribution_container_form.contribution_container.id),
-              notice: t(".success_html", link: gobierto_participation_process_contribution_container_url(@contribution_container_form.contribution_container.slug,
-                                                                                                                 process_id: @process.slug,
-                                                                                                                 host: current_site.domain))
+              edit_admin_participation_process_contribution_container_path(
```

Comment:
I think you forgot the the host to the options, so the domain of the URL will be wrong. Could you check ?

Reply:
Yes, I added the domain, sorry.